arbnode/transaction_streamer.go

```diff
@@ -988,14 +1019,64 @@ func (s *TransactionStreamer) WriteMessageFromSequencer(
 		BlockHash:       &msgResult.BlockHash,
 	}
 
+	l1Block := msgWithMeta.Message.Header.BlockNumber
+
+	if s.config().SovereignSequencerEnabled && s.espressoClient != nil && wavmio.IsHotShotLive(l1Block) {
```

Comment:
We can't use `wavmio` here. `wavmio` is the interface for the `replay wasm` getting information. 

please see: https://www.notion.so/espressosys/Arbitrum-Fraud-Proofs-50dfc4f141434475ad23c34c8d37bd9e

Reply:
This makes sense! Maybe we can just check the hotshot liveness here. So the escape hatch is still compatible with the current design.

Reply:
Based on our discussion in standup, will remove this!